src/verify.c

```diff
@@ -785,6 +785,33 @@ static int find_unsafe_to_delete(const void *a, const void *b)
 	return -1;
 }
 
+static bool init_tracking_dir(const char *state_dir, const char *init_file)
+{
+
+	char *tracking_dir;
+	char *tracking_file;
+
+	/* make sure the state directory exist */
+	if (create_state_dirs(state_dir)) {
+		return false;
+	}
+
+	/* add a temporary control file to the tracking
+	 * directory so it is not empty to avoid tracking
+	 * all installed bundles */
+	tracking_dir = mk_full_filename(state_dir, "bundles");
+	tracking_file = mk_full_filename(tracking_dir, init_file);
+	int fd = open(tracking_file, O_RDWR | O_CREAT, S_IRUSR | S_IWUSR);
```

Comment:
I needed to use a temporary file because when installing a new system the state dir will always be empty, so all bundles in /usr/share/clear/bundles are going to be copied as tracked every time, not only the ones requested by --bundles. So it is a hack to make the track_installed() function believe the tracking directory is not empty and only copy those specified.

Reply:
you can touch os-core instead. It's always required to be tracked.

Reply:
Actually not. Leave the way it is